src/dataflow/components/dataflow-program.tsx

```diff
@@ -195,6 +197,16 @@ export class DataflowProgram extends BaseComponent<IProps, IState> {
     }
   }
 
+  private getEditorStyle = () => {
+    const style: React.CSSProperties = {};
+    const documentElts = document.getElementsByClassName("document-content");
+    const documentElt = documentElts && (documentElts[0]);
+    const titlebarElts = document.getElementsByClassName("other-doc titlebar");
```

Comment:
```suggestion
    const titlebarElts = document.querySelector(".document .titlebar");
```
🤔 I just eliminated the `other-doc` class in my most recent titlebar styling PR on master.

Reply:
Ah, good to know!  That class name seemed a little odd.

Reply:
Internally, personal documents and learning log documents are essentially identical and so there are a bunch of functions that handle both of them such as `createOtherDocument()`, `copyOtherDocument()`, etc. But it turned out not to be useful as a class name because personal documents and learning logs are styled differently.

Reply:
Fixed.